### PR TITLE
Allow serial access to file in cpg_open

### PR DESCRIPTION
**Pull request by TonyGarratt**

### CODE REVIEW

**Comment by MicK7** [comment]
Please group this merge request with #102

File: src/pcgnslib.c [hunk 1]
```diff
@@ -421,11 +421,19 @@ int cgp_open(const char *filename, int mode, int *fn)
 
     /* Flag is true if MPI_Init or MPI_Init_thread has been called and false otherwise. */
     pcg_mpi_initialized = 0;
-    /* check if we are actually running a parallel program */
-    MPI_Initialized(&pcg_mpi_initialized);
-
-    /* Flag this as a parallel access */
-    strcpy(hdf5_access,"PARALLEL");	
+    if (mode < 10) {
```
Comment (brtnfld) [inline src/pcgnslib.c]: Should use the variable instead of a hard-coded number.

Comment (TonyGarratt) [reply src/pcgnslib.c]: Good point. Let me fix that

**Comment by TonyGarratt** [comment]
Has been merged into #105

**Pull request closed without merging**